ui/app/templates/components/secret-edit.hbs

```diff
@@ -27,7 +27,16 @@
     <div class="level-right is-marginless">
       <div class="field is-horizontal is-flex-end is-single-line">
         <div class="control is-flex">
-          <input data-test-secret-json-toggle=true id="json" type="checkbox" name="json" class="switch is-rounded is-success is-small" checked={{showAdvancedMode}} onchange={{action "toggleAdvanced" value="target.checked"}} />
+          <input
+            data-test-secret-json-toggle=true
+            id="json"
+            type="checkbox"
+            name="json"
+            class="switch is-rounded is-success is-small"
+            checked={{showAdvancedMode}}
+            onchange={{action "toggleAdvanced" value="target.checked"}}
+            disabled={{and (eq mode 'show') secretDataIsAdvanced}}
```

Comment:
This line is the only code change - just formatted the input for easier reading.

Reply:
thank you for the formatting.